webui/react/src/components/kit/Avatar.tsx

```diff
@@ -47,34 +52,51 @@ export const getColor = (name = '', darkLight: DarkLight): string => {
 };
 
 const Avatar: React.FC<Props> = ({
-  displayName,
+  text,
   hideTooltip,
   noColor,
-  size = Size.Medium,
+  size = Size.Small,
   square,
+  textColor = 'white',
```

Comment:
Can we infer text color from the background color (and the theme)?

Reply:
this actually isn't theme or background color dependent. we always used white for users and always used black for workspaces.

Reply:
in light of the saturation/lightness issue you noticed, I'm updating this to be a `palette` instead of just `textColor`.